dolphinscheduler-ui/src/locales/en_US/project.ts

```diff
@@ -308,6 +308,8 @@ export default {
     task_group_name: 'Task group name',
     task_group_queue_priority: 'Priority',
     number_of_failed_retries: 'Number of failed retries',
+    cpu_quota: 'Cpu quota',
```

Comment:
It seems to be better that the cpu be capitalized.

Reply:
I'll fix it.